no-lstdcxxfs.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/openrct2/CMakeLists.txt b/src/openrct2/CMakeLists.txt
+index b0bd3e9..e937b45 100644
+--- a/src/openrct2/CMakeLists.txt
++++ b/src/openrct2/CMakeLists.txt
+@@ -25,7 +25,7 @@ SET_CHECK_CXX_FLAGS(${PROJECT_NAME})
+ # GCC / Clang likes us to pass the -lstdc++fs flag to link C++17 filesystem implementation.
+ if (NOT MINGW AND NOT ${CMAKE_SYSTEM_NAME} MATCHES "FreeBSD")
+     if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU" OR CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+-        target_link_libraries(${PROJECT_NAME} stdc++fs)
++        #target_link_libraries(${PROJECT_NAME} stdc++fs)
```

Comment:
How does it link with the lib removed?

Reply:
Quoting GCC 9 change log:

```
Using the types and functions in <filesystem> does not require linking with -lstdc++fs now. 
```

Reply:
And in the build environment, it seems that you can't use `-lstdc++fs` - I got an error, which prompted this patch.

Reply:
Interesting, we've been using such compilers without issues.

The quoted wording does not imply it's forbidden to link, so I wonder why the change and how best to detect it. I wonder how e.g. OpenMandriva works.

Can you follow up with patch upstream so we can test it?

Reply:
I suspect it's dependent on the distro. I think freedesktopsdk team decided to drop `libstdc++fs.a` from their GCC based on that change log entry. Arch still ships it as well; I'll ask around.

Reply:
Happy to follow up - is an issue OK? I don't really know how to do anything more sophisticated than patching the line out entirely.